### DISCUSSION TITLE
increase buffer size?

### DISCUSSION THREAD

**Discussion by dblondin** [melanchall/drywetmidi#150] [Q&A] [answered]
Hi this is working great in Unity. Really psyched this is available. I have one issue I've not been able to sort out. Comparing unity receiving a thick stream of MIDI data agains some hardware receiving the same, my Unity project seems to be dropping events while my hardware keeps up fine. Is there a way to increase the size of the buffer in Unity so that there's less chance of events getting dropped?

**Comment by melanchall** [comment, marked as answer]
Hi,

What buffer are you talking about? Unfortunately I can't help you with Unity because my library is not about Unity. It can be used there (and in many other environments) but I'm a developer of the DryWetMIDI and can assist with it only. If the problem related with DryWetMIDI, please, provide more info. What do you mean _Unity project dropping events_? You use `InputDevice` and not all events are received? What events are not received? Show you code please and provide more information.
